.github/actions/with-post-step/action.yml

```diff
@@ -0,0 +1,42 @@
+# ========================================================================= #
```

Comment:
is it supposed to be used anywhere?

Reply:
I'll see if we still need it. Wanted to use in a single `common` action, but didn't work out